### PR TITLE
Add coverage for content view filter delete in CLI

### LINKED ISSUE #2461: Automate : content-view filter delete in cli
please automate:
$ hammer content-view filter delete --help
Usage:
    hammer content-view filter delete [OPTIONS]

Options:
 --content-view CONTENT_VIEW_NAME        Content view name
 --content-view-id CONTENT_VIEW_ID       content view numeric identifier
 --id ID                                 filter identifier
 --name NAME                             Name to search by
 --organization ORGANIZATION_NAME        Organization name to search by
 --organization-id ORGANIZATION_ID       organization ID
 --organization-label ORGANIZATION_LABEL Organization label to search by
 -h, --help                              print help

### PR DESCRIPTION
**Pull request by oshtaier**
Closes #2461

As a bonus, adding update to docstring suggested by @sthirugn in my last PR (don't want to create separate PR :smile_cat:)

```
nosetests tests/foreman/cli/test_contentviewfilter.py -m test_delete_cvf
..........
----------------------------------------------------------------------
Ran 10 tests in 296.903s

OK
```

### CODE REVIEW

File: robottelo/ui/computeprofile.py [hunk 1]
```diff
@@ -14,7 +14,13 @@ def search(self, name):
         return self.search_entity(name, locators['profile.select_name'])
 
     def select_resource(self, profile_name, res_name, res_type):
-        """Select necessary compute resource from specific compute profile"""
+        """Select necessary compute resource from specific compute profile
+        :param profile_name: Name of profile that contains required compute
+        resource (e.g. '2-Medium' or '1-Small')
+        :param res_name: Name of compute resource to select from the list
+        :param res_type: Type of compute resource (e.g. 'Libvirt' or 'Docker')
```
Comment (Ichimonji10) [inline robottelo/ui/computeprofile.py]: Easy fix.

    """Select necessary compute resource from specific compute profile

    :param profile_name: Name of profile that contains required compute
        resource (e.g. '2-Medium' or '1-Small')
    :param res_name: Name of compute resource to select from the list
    :param res_type: Type of compute resource (e.g. 'Libvirt' or 'Docker')

    """

**Comment by Ichimonji10** [comment]
ACK pending comment.

File: tests/foreman/cli/test_contentviewfilter.py [hunk 2]
```diff
@@ -806,3 +806,129 @@ def test_update_cvf_with_new_repo_negative(self):
...
+        self.assertEqual(result.return_code, 0)
+        self.assertEqual(len(result.stderr), 0)
+
+        result = ContentView.filter_info({
+            u'content-view-id': self.content_view['id'],
+            u'name': self.cvf_name,
+        })
+        self.assertNotEqual(result.return_code, 0)
+
+    def test_delete_cvf_with_name_negative(self):
```
Comment (sthirugn) [inline tests/foreman/cli/test_contentviewfilter.py]: This test doesn't make sense to me.  Can you explain what you are trying to do?

Comment (oshtaier) [inline tests/foreman/cli/test_contentviewfilter.py]: I am trying to delete non-existent content view filter using generated name

File: tests/foreman/cli/test_contentviewfilter.py [hunk 3]
```diff
@@ -806,3 +806,127 @@ def test_update_cvf_with_new_repo_negative(self):
...
+        """Test: Try to delete non-existent filter using generated name
+
+        @Feature: Content View Filter
+
+        @Assert: System returned error
+
+        """
+        result = ContentView.filter_delete({
+            u'content-view-id': self.content_view['id'],
+            u'name': self.cvf_name,
```
Comment (sthirugn) [inline tests/foreman/cli/test_contentviewfilter.py]: Although I like this to be the following, this is non blocker.
```u'name': u'invalid'```

**Comment by sthirugn** [comment]
ACK

**Comment by oshtaier** [comment]
Ready to merge :)

**Pull request merged**